socorro/external/boto/crash_data.py

```diff
@@ -49,7 +49,7 @@ def get(self, **kwargs):
             # Raw Crash
             "meta": "get_raw_crash",
             # Redacted processed crash
-            "processed": "get_processed",
+            "processed": "get_processed_crash",
```

Comment:
We had "get_raw_crash" and "get_processed'. I made that consistent everywhere so now we have "get_raw_crash" and "get_processed_crash".